cfgov/cfgov/urls.py

```diff
@@ -95,39 +96,96 @@ def empty_200_response(request, *args, **kwargs):
     url(r'^owning-a-home/closing-disclosure/$',
         TemplateView.as_view(
             template_name='owning-a-home/closing-disclosure/index.html'),
-            name='closing-disclosure'
-    ),
+        name='closing-disclosure'),
     url(r'^owning-a-home/explore-rates/',
         TemplateView.as_view(
             template_name='owning-a-home/explore-rates/index.html'),
-            name='explore-rates'
-    ),
+        name='explore-rates'),
     url(r'^owning-a-home/loan-estimate/$',
         TemplateView.as_view(
             template_name='owning-a-home/loan-estimate/index.html'),
-            name='loan-estimate'
-    ),
+        name='loan-estimate'),
 
-    # Temporarily serve Wagtail OAH journey pages at `/process/` urls.
-    # TODO: change to redirects after 2018 homebuying campaign.
+    # Temporarily serve Wagtail OAH journey pages at `/process/` urls
+    # TODO: change to redirects after 2018 homebuying campaign
     url(r'^owning-a-home/process/(?P<path>.*)$',
         lambda req, path: ServeView.as_view()(
             req, 'owning-a-home/{}'.format(path or 'prepare/')
         ),
     ),
     # END TODO
 
+    url(r'^how-to-prepare-your-finances-for-buying-a-home/',
```

Comment:
Is there any particular reason why these are being added as redirects here instead of in Apache?

Reply:
I added redirects into urls.py as there were already redirects in this file, and I could easily verify the redirects were working locally using a Docker container. When I found out there was a redirects.conf for Apache, I put the redirects into redirects.conf. Would you like me to remove the redirects I added from the urls.py file?

Reply:
Yes, thanks. Generally if we are creating permanent redirects that don't require any Python logic, they should be placed in the Apache redirects file. Sometimes we'll put redirects into the urls.py temporarily, or initially behind a feature flag that gets removed later, and neglect to move that now-permanent redirect to Apache.

Reply:
Thanks for clarifying and taking time to review. I will remove permanent redirects that don't require any Python logic from urls.py.